src/index.js

```diff
@@ -10,6 +10,8 @@ const PsModuleMap = require('./PsModuleMap');
 const Psc = require('./Psc');
 const PscIde = require('./PscIde');
 const dargs = require('./dargs');
+const execSync = require('child_process').execSync
```

Comment:
Just to ensure compatibility can we change this to `cross-spawn`? We run `psc`, etc., already with `cross-spawn`.